op-batcher/db/history_db.go

```diff
@@ -4,67 +4,42 @@ import (
 	"encoding/json"
```

Comment:
delete the history files? IIRC they aren't used anymore

Reply:
done